.github/workflows/CI.yml

```diff
@@ -56,15 +56,25 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
+    - name: Create and start virtual environment
+      run: |
+        python3 -m venv venv
+        source venv/bin/activate
```

Comment:
What's the point of this when everything is being run in docker?

Reply:
Removed this step